fastlane/Fastfile

```diff
@@ -106,6 +106,9 @@ platform :ios do
 
     ios_bump_version_release
     new_version = ios_get_app_version
+    extract_release_notes_for_version(version: new_version,
+      release_notes_file_path: File.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
+      extracted_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'))
```

Comment:
Notice that one uses `ENV` and the other the `PROJECT_ROOT_FOLDER`. I should have properly used the same way to read the value for consistency but I realized it only after making the commit.

I'll followup with a PR that removes all accesses of `ENV["PROJECT_ROOT_FOLDER"]` in favor of using the constant, which is our desired approach.